htdocs/helpFiles/Entering-Essays.html

```diff
@@ -0,0 +1,12 @@
+<h2 style="text-align: center">Entering Essays</h2>
+
+<p>
+	In an essay answer text box, you can type a long answer. After after you hit submit, it will be saved so that your
+	instructor can read it and assign a score at a later time.
+</p>
+<p>If your instructor makes any comments on your answer, those comments will appear on the exercise page.</p>
+<p>
+	You can use the MathQuill equation editor to make your math content look pretty. Once you have constructed a math
+	expression, click the Insert button. In the answer text box, your math content will look like code, but the Preview
+	button will show you what your isntructor will see.
+</p>
```

Comment:
I would also mention that latex can be directly entered using `\( ... \)`, or use the equation editor to insert formulas. Could be useful for users who know a little latex but unsure about how to insert it.